target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -247,6 +247,10 @@ iodata,wn-ac733gr3)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$boardname:green:wlan5g" "phy0radio"
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "$boardname:green:wlan2g" "phy1radio"
 	;;
+iptime,a8004t)
+	ucidef_set_led_wlan "wlan2g" "WiFi 2.4GHz" "$boardname:orange:wlan2g" "phy0radio"
+	ucidef_set_led_wlan "wlan5g" "WiFi 5GHz" "$boardname:orange:wlan5g" "phy1radio"
```

Comment:
Have you tried whether you can use triggers in DTS like
linux,default-trigger = "phy0tpt";

e.g. in https://github.com/openwrt/openwrt/commit/b7ec6659a14d1a11e52b56235274b3e46d22e331